### PR TITLE
feat: Multi-sign headways UI

### PR DESCRIPTION
**Pull request by fjlanasa**
#### Summary of changes
**Asana Tickets:** 
 [😷 PIOs can set headways for multiple signs at a time](https://app.asana.com/0/629710038948323/1166224048806859/f)
* [create accordion feature to enable multi-sign headway](TICKET_LINK)
* [allow editing of multi-sign headways ](https://app.asana.com/0/584764604969369/1166302267970961/f)
* [signs-ui headway form per line, with fields for headways during several time periods](https://app.asana.com/0/584764604969369/1167184327376438/f)

Screen Shot 2020-03-23 at 9 19 56 AM

#### Reviewer Checklist
- [ ] Meets ticket's acceptance criteria
- [ ] Any new or changed functions have typespecs
- [ ] Tests were added for any new functionality (don't just rely on Codecov)
- [ ] This branch was deployed to the staging environment and is currently running with no unexpected increase in warnings, and no errors or crashes (compare on Splunk: [staging](https://mbta.splunkcloud.com/en-US/app/search/search?q=search%20index%3Dsigns-ui-dev%20%22%5Berror%5D%22%20OR%20%22%5Bwarn%5D%22%20OR%20%22CRASH%22&display.page.search.mode=verbose&dispatch.sample_ratio=1&earliest=-4h%40m&latest=now&sid=1545840769.3874970) vs. [prod](https://mbta.splunkcloud.com/en-US/app/search/search?q=search%20index%3Dsigns-ui-prod%20%22%5Berror%5D%22%20OR%20%22%5Bwarn%5D%22%20OR%20%22CRASH%22&display.page.search.mode=verbose&dispatch.sample_ratio=1&earliest=-4h%40m&latest=now&sid=1545840745.3874956))

### CODE REVIEW

**Comment by losvedir** [comment]
Just a note from playing around with signs-ui-dev for my branch: I think we shouldn't have the headway configuration on the SL3 tab, since we don't actually control those signs anymore.

**Comment by losvedir** [comment]
I think we might want to remove the accordion's expand/collapse little up/down arrow functionality, since after the initial values are set, we can never interact with that control again.

File: assets/js/ConfiguredHeadwaysForm.jsx [hunk 1]
```diff
@@ -0,0 +1,191 @@
...
+        <h5 className="viewer--configured-headways-form-header">Headways</h5>
+      </div>
+      <Formik
+        validationSchema={formSchema}
+        enableReinitialize
+        initialValues={initialFormValues}
+        onSubmit={handleSubmit}
+      >
+        {({
+          dirty, isValid, resetForm,
```
Comment (losvedir) [inline assets/js/ConfiguredHeadwaysForm.jsx]: I don't think I've seen this before where in the JSX there's a function. I know that JSX elements like `<div>` compile to a function call of some sort, but what does it mean for a function to be here? Does it get called every render? Is this something specific to `Formik`?

Comment (fjlanasa) [reply assets/js/ConfiguredHeadwaysForm.jsx]: Basically the `Formik` component is setup to handle the `children` prop being a function (essentially a [render prop](https://reactjs.org/docs/render-props.html) or a JSX Element, calling the provided function within its own [`render`](https://github.com/jaredpalmer/formik/blob/master/packages/formik/src/Formik.tsx#L997) function. This isn't a default behavior of react, but I've seen a few libraries take this approach to passing props (usually internal state) from a parent component to a child.

Comment (fjlanasa) [reply assets/js/ConfiguredHeadwaysForm.jsx]: The alternative is the `Formik` component's `component` prop, which is also supposed to be a function (taking the props passed down by `Formik` as an arugment) and returning JSX.

https://jaredpalmer.com/formik/docs/api/formik#component-reactcomponenttypeformikpropsvalues

**Pull request merged**